examples/presigned_post_policy.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # MinIO Python Library for Amazon S3 Compatible Cloud Storage,
-# (C) 2015 MinIO, Inc.
+# (C) 2023 MinIO, Inc.
```

Comment:
License could be `2015` to `2023`

Reply:
Ok @balamurugana it make sense, I have changed this accordingly